packages/aragon-cli/src/util.js

```diff
@@ -302,6 +302,15 @@ const parseStringIfPossible = target => {
   return target
 }
 
+/**
+ * Validates an Aragon ENS subdomain name
+ * @param {string} name Name
+ * @returns {boolean} `true` if valid
+ */
+function isValidEnsName(name) {
+  return /^[a-z0-9]+$/.test(name)
```

Comment:
@macor161 ENS allow the use of the `-` symbol as well.

Reply:
Yes I was wondering whether to add the hyphen or not, since it is not accepted in the main onboarding app (app.aragon.org). Is there an official documentation where I can find the accepted chars for Aragon subdomains?

Reply:
You are right about that. Nevertheless in that cause those are to link a dao to an [`aragonId`](https://github.com/aragon/aragon-id). We are creating a subdomain in the `open.aragonpm.eth` registrar for Aragon apps. You think we should also stick to not use the hyphen? 

Sorry I went ahead and made the changes 😅

Reply:
The choice to not use the hyphen is a purely arbitrary decision made by us, and not an ENS limitation. You can use any [IRI](https://www.ietf.org/rfc/rfc3987.txt) (including emojis, etc.).

The frontend actually accepts [this regex](https://github.com/aragon/aragon/blob/master/src/web3-utils.js#L170) for self-assigned ENS names. Note that again, it is still a subset of possible ENS names, but it was chosen to avoid potential spoofing attacks on the frontend with unicode characters.

Reply:
@sohkai Thanks for clarifications! 
@0xGabi Thanks for those light-speed fixes :) 
So after discussing the subject, we decided to keep the hyphen since it was already possible to create apps that contain them with the current CLI.